sympy/integrals/integrals.py

```diff
@@ -396,6 +397,8 @@ def doit(self, **hints):
             return S.Zero
 
         # now compute and check the function
+        if isinstance(self.function, Sum):
+            return Sum(Integral(self.function.function, self.variables[0]).doit(), self.function.limits)
```

Comment:
In general, summation and integration ops are *not* commutative.  I.e. mathematically, this code is wrong.

Unsubscribed.